src/components/SEP/Proposals/SEPProposalsAndAssignmentsTable.tsx

```diff
@@ -134,6 +140,14 @@ const SEPProposalsAndAssignmentsTable: React.FC<SEPProposalsAndAssignmentsTableP
                   assignedReviewer.sepMemberUserId
               ) || [];
 
+            // console.log(
```

Comment:
Do we need to keep this commend or it can be removed?

Reply:
I'll do a cleanup later as there are remnants of code that I used for debugging or not needed.